code/modules/library/lib_items.dm

```diff
@@ -88,7 +88,7 @@
 		else
 	return
 
-/obj/structure/bookcase/update_icon()
+/obj/structure/fcase/update_icon()

```

Comment:
Why this has been changed? Also, if this is mapped somewhere, it might conflict or something?